humility-doppel/src/lib.rs

```diff
@@ -318,10 +318,29 @@ impl humility::reflect::Load for TaskId {
 pub struct RingbufEntry {
     pub line: u16,
     pub generation: u16,
-    pub count: u32,
+    pub count: RingbufCount,
     pub payload: Value,
 }
 
+#[derive(Clone, Debug)]
+pub struct RingbufCount(pub u32);
+
+impl humility::reflect::Load for RingbufCount {
+    fn from_value(v: &Value) -> Result<Self> {
+        match v.as_base()? {
+            Base::U16(v) => Ok(Self(u32::from(*v))),
+            Base::U32(v) => Ok(Self(*v)),
+            _ => bail!("not a u32 or u16: {v:?}"),
```

Comment:
Take it or leave it: should we just go ahead and support any integer here, as well? Just in case we start using `u8` or `u64` counters later?[^1]

[^1]: This is probably unlikely, but...be conservative in what you admit, but liberal in what you accept?

Reply:
Sure, I bumped the internal size up to a `u64` and made it accept any unsigned integer (bdd39928cc7bae70e192085f844f1b7d1b443ab7)